docs/src/Checkbox.doc.js

```diff
@@ -85,64 +93,35 @@ card(
     name="Example: Accessibility"
     defaultCode={`
 function CheckboxExample() {
-  const [checked, setChecked] = React.useState(true);
+  const [checkedSm, setCheckedSm] = React.useState(true);
+  const [checkedMd, setCheckedMd] = React.useState(true);
+
   return (
-    <Box alignItems="center" direction="row" display="flex">
+    <Box margin={-2}>
+      <Box padding={2}>
       <Checkbox
-        checked={checked}
-        id="usa"
-        name="usa"
+        checked={checkedSm}
+        id="usaSm"
+        label="United States of America"
+        name="usaSm"
         onChange={({ checked }) => {
-          setChecked(checked);
+        setCheckedSm(checked);
         }}
+        size='sm'
       />
-      <Label htmlFor="usa">
-        <Box paddingX={2}>
-          <Text>United States of America</Text>
-        </Box>
-      </Label>
-    </Box>
-  );
-}
-`}
-  />
-);
-
-card(
-  <Example
-    description="
-    If you have lots of checkboxes you can stack them on top of one another.
```

Comment:
@AlbertCarreras could we at least have one example of multiple checkboxes on top of another? E.g. how much padding should there be in between them?

Reply:
DONE